.github/workflows/upgradeable.yml

```diff
@@ -29,4 +30,6 @@ jobs:
       - run: bash scripts/git-user-config.sh
       - name: Transpile to upgradeable
         run: bash scripts/upgradeable/transpile-onto.sh ${{ github.ref_name }} origin/${{ github.ref_name }}
+        env:
+          REMOTE: https://github.com/${{ github.repository }}.git
```

Comment:
I'm not sure REMOVE it the best name. It should reflect that this is a dependency/submodule/peer project

Reply:
I agree it's not clear in this `.yml` because it lacks of context. `SUBMODULE_REMOTE` should be better imo

Reply:
Renamed.